### PR TITLE
Add PTIX Support

### PR DESCRIPTION
**Pull request by trgibeau**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -20,7 +20,9 @@ jobs:
     - name: Setup .NET Core
       uses: actions/setup-dotnet@v2
       with:
-        dotnet-version: '8.x'
+        dotnet-version: |
+          6.x
```
Comment (ivberg) [inline .github/workflows/ci.yml:24]: Any reason why 6.x is needed now? It's not clear from the other changes why?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -47,6 +49,28 @@ jobs:
     - name: Test projects
       run: dotnet test Microsoft-Perf-Tools-Linux-Android.sln --no-restore --verbosity normal
 
+    - name: Install PluginTool
+      run: dotnet tool install --global Microsoft.Performance.Toolkit.Plugins.Cli --version 0.1.25-preview      
+
+    - name: Create PTIX
+      run: |
+        $dirs = @{
```
Comment (ivberg) [inline .github/workflows/ci.yml:57]: These seems the same as below. Ideally we should check to see if the dirs can be shared between different run cmds

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -47,6 +49,28 @@ jobs:
...
+          "LTTngDataExtensions" = "LTTngDataExtensions/bin/Release/netstandard2.1/"
+          "LTTngDriver" = "LTTngDriver/bin/Release/netcoreapp8.0/"
+          "Perfetto" = "PerfettoCds/bin/Release/netstandard2.1/"
+          "Cloud-init" = "LinuxLogParsers/LinuxPlugins-MicrosoftPerformanceToolkSDK/Cloud-init/bin/Release/netstandard2.1/"
+          "Dmesg" = "LinuxLogParsers/LinuxPlugins-MicrosoftPerformanceToolkSDK/DmesgIsoLog/bin/Release/netstandard2.1/"
+          "WaLinuxAgent" = "LinuxLogParsers/LinuxPlugins-MicrosoftPerformanceToolkSDK/WaLinuxAgent/bin/Release/netstandard2.1/"
+          "AndroidLogCat" = "LinuxLogParsers/LinuxPlugins-MicrosoftPerformanceToolkSDK/AndroidLogCat/bin/Release/netstandard2.1/"
+        }
+
+        foreach ($dir in $dirs.Keys) {
```
Comment (ivberg) [inline .github/workflows/ci.yml]: The .ptix files also need to be copied to GH Workspace like the binary files are copied

**Review by ivberg — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (trgibeau) [reply .github/workflows/ci.yml:24]: The plugin tool currently targets .net 6, we will be pushing out a new version in the near future that is .net 8

File: .github/workflows/ci.yml (hunk 2 above)
Comment (trgibeau) [reply .github/workflows/ci.yml:57]: They only share the plugins, the one below has a couple extra folders.  

I noticed these are also duplicated above in the dotnet build command.  

I'm wondering if the entire YML file needs to be rewritten, but that would be better to have in a different PR

**Review by ivberg — APPROVED** [review]

**Pull request merged**